app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntryUpdate.kt

```diff
@@ -20,12 +20,23 @@ class CustomDrugEntryUpdate : Update<CustomDrugEntryModel, CustomDrugEntryEvent,
       is FrequencyEdited -> next(model.frequencyEdited(event.frequency), SetDrugFrequency(event.frequency), SetSheetTitle(model.drugName, model.dosage, event.frequency))
       is AddMedicineButtonClicked -> createOrUpdatePrescriptionEntry(model)
       is CustomDrugSaved -> dispatch(CloseBottomSheet)
-      is CustomDrugFetched -> drugFetched(event.prescription)
+      is CustomDrugFetched -> drugFetched(model, event.prescription)
     }
   }
 
-  private fun drugFetched(prescription: PrescribedDrug): Next<CustomDrugEntryModel, CustomDrugEntryEffect> {
-    return dispatch(CloseBottomSheet)
+  private fun drugFetched(
+      model: CustomDrugEntryModel,
+      prescription: PrescribedDrug
+  ): Next<CustomDrugEntryModel, CustomDrugEntryEffect> {
+    return if (prescription.isDeleted) {
```

Comment:
I'm curious. In what scenario would we open a deleted `PrescribedDrug` here? We don't show deleted drugs in the UI, so how would the nurse open this screen with the UUID of a deleted `PrescribedDrug` record?

Reply:
Yeah, this is not needed now as the deletion of the custom drug is explicit.